### ISSUE TITLE
Content Authors -> Writers ?

### ISSUE THREAD

**Issue by shawna-slh** [w3c/wai-arrm#12]
WCAG uses "authors" for everyone creating content, including designers and developers. (whether that was a good decision or not is moot at this point)

Therefore, for the ARRM roles, consider "Writers" instead of "Content authors".

Maybe this was already discussed? If do, hopefully it's easy to point to the GitHub issue(s) and/or minute(s).

**Comment by shawna-slh** [comment]
One issue, though, is that wouldn't include content designers as well. Maybe Content Creators?

**Comment by AshJ892** [comment]
I have concerns about using "Content Creators" to replace "Content Authors" for a few reasons

1. Content Creators are quite commonly associated as its own job role as a Social Media Content Creator (such as a Youtuber, or an Instagram influencer, etc.), which could make it not so immediately clear to some people that we are talking about a broad range of jobs.
2. Content Creators are listed as an example job title under the Content Authors section - it feels a bit off to say that an example of a Content Creator (meaning: range of different jobs) is a Content Creator (meaning: a more specific job title).

**Comment by mgifford** [comment]
Having a clarification statement might be useful, so that we could say, "When we say Writer, we also want to include Content Strategist, Content Creator, Content Designer, Content Author, Digital Copywriter, Content Producer, Video Producer, Podcast Host, etc. "

Also, Author is perhaps more inclusive than Writer.

**Comment by dboudreau** [comment]
@AshJ892 Brings up a good point about the connotations related to "content creators" and social media. For that reason alone, I would personally avoid going down the path of content creators.

**Comment by anna-yeddi** [comment]
I agree with @AshJ892  and @dboudreau about "content creators" use. And I think it would be clearer to keep the "content" word in the term so it is clearly separated from the UX and engineering professionals. 

WDYT about `content writers`? It is generally described as:
> A content writer is a professional responsible for creating written materials for various online and offline platforms, with the goal of engaging target audiences, promoting brands, or providing informative content
For my (non-content related professional eye) it may seem wide enough for our case?

**Comment by dboudreau** [comment]
Do we all agree on Content writers as a general label?

**Comment by dboudreau** [comment]
Not having hear from anyone about this since my last comment/question... 
@mgifford what would it take for us to standardize to "content writers" and close this issue?

**Comment by mgifford** [comment]
There aren't too many instances in the files:

**3 files "content creator" -> "content writer"**
https://github.com/search?q=repo%3Aw3c%2Fwai-arrm+%22content+creator%22&type=code

**8 files "content author" -> "content writer"**
https://github.com/search?q=repo%3Aw3c%2Fwai-arrm+3+%22content+author%22&type=code

_Leaving "content publish" will stay the same._ 
https://github.com/search?q=repo%3Aw3c%2Fwai-arrm+3+%22content+publish%22&type=code

_Leave "content creation" as well._
https://github.com/search?q=repo%3Aw3c%2Fwai-arrm+%22content+creation%22&type=code

**Comment by shawna-slh** [comment]
+1 to "content creator" -> "content writer"
